### PR TITLE
changed link colour var to use specific link var rather than primary

### PR DESCRIPTION
**Pull request by cjstevens78**

### CODE REVIEW

**Review by andybroomfield — CHANGES_REQUESTED** [review]
The first part is working as expected, however the hover / focus colour is not working becuase it is overidden by the following in _shame.scss.
````scss
.access-background { 
  background-color: $focus;
  color: $secondary;
  text-decoration: none;
  box-shadow: 0 0 0 0.125rem $focus;
}
````

File: assets/scss/lib/partials/_shame.scss [hunk 1]
```diff
@@ -30,16 +30,6 @@ table {
   }
 }
 
-// // remove left hand border on google search results causing content to indent on hover
```
Comment (cjstevens78) [inline assets/scss/lib/partials/_shame.scss:33]: removed - croydon specific sass

File: assets/scss/lib/partials/_shame.scss [hunk 2]
```diff
@@ -30,16 +30,6 @@ table {
...
-// .gsc-webResult.gsc-result {
```
Comment (cjstevens78) [inline assets/scss/lib/partials/_shame.scss:34]: removed - croydon specific sass

File: assets/scss/lib/partials/_shame.scss [hunk 3]
```diff
@@ -50,19 +40,6 @@ iframe {
   }
 }
 
-// important:: fix form.io overriding this specific pseudo in the footer
```
Comment (cjstevens78) [inline assets/scss/lib/partials/_shame.scss:53]: removed - croydon specific sass

**Review by andybroomfield — APPROVED** [review]
This works for me and happy to see this changed merged.

**Pull request merged**